build/spas.js

```diff
@@ -31,10 +31,8 @@ const contributorSpotlightRoot = CONTRIBUTOR_SPOTLIGHT_ROOT;
 
 let featuredContributor;
 
-async function buildContributorSpotlight(options) {
-  // for now, these will only be available in English
-  const locale = "en-US";
-  const prefix = `/${locale}/community/spotlight`;
+async function buildContributorSpotlight(options, locale) {
+  const prefix = "community/spotlight";
```

Comment:
I haven't seen a lot of relative paths in our code base, so I would opt for an absolute path, which is also more clearer to readers (of the code):

```suggestion
  const prefix = `/${locale}/community/spotlight`;
```

Reply:
Nope, my suggestion is exactly what caused the locale to be added twice to the output path below. Not good.